docker-compose.yml

```diff
@@ -1,26 +1,21 @@
 # Use postgres/example user/password credentials
-version: '3.1'
+# When using 'docker-compose up' it's possible that the Flyway container starts before the database is up, causing an error
+# In this case just do a relaunch.
+# TODO: integrate wait-for-it.sh or something to wait until DB is up before running Flyway
```

Comment:
I think it will more convenient to utilize health check on DB:
- https://docs.docker.com/compose/compose-file/#healthcheck

Reply:
Sadly, `docker-compose` doesn't support the `condition` clause on `depends_on` section since version 3, so it doesn't seem to be applicable here.
See https://docs.docker.com/compose/compose-file/#depends_on